Makefile.toml

```diff
@@ -2,13 +2,24 @@
 command = "cargo"
 args = ["check", "--tests", "--benches", "--all-targets"]
 
+[tasks.install-nightly]
+command = "rustup"
+args = ["toolchain", "install", "nightly"]
+
+[tasks.install-nightly-rustfmt]
+dependencies = ["install-nightly"]
+command = "rustup"
+args = ["component", "add", "rustfmt", "--toolchain", "nightly"]
+
 # -- CI Tasks --
 [tasks.ci]
 dependencies = ["ci-format", "ci-clippy", "ci-build", "ci-docs", "ci-doc-test", "ci-test"]
 
 [tasks.ci-format]
+dependencies = ["install-nightly-rustfmt"]
```

Comment:
Looks good!

The main change I think we should make is installing Rust nightly through a GitHub action instead of through cargo make. The cargo make can then just assume Rust nightly is installed. That would go [here](https://github.com/boltlabs-inc/tss-ecdsa/blob/25b7c711255a94eb34e6f5d05ef0cf130b7989e0/.github/workflows/main.yml#L19-L27).

We can just do it the same way [our other project does it](https://github.com/boltlabs-inc/key-mgmt/blob/201061a37e214b1ef0786b22e6de3707b55e140e/.github/workflows/rust.yml#L28).